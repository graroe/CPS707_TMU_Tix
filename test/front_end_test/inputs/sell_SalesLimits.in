login
sales
sell
CirqueDuSoleil
9
!q
logout
login
admin
sell
CirqueDuSoleil
10
logout