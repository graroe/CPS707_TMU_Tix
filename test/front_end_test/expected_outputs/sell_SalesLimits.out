Enter session type <sales or admin>: 
Sales agent login successful.
Enter event name: 
Current number of tickets: 10\nEnter number of tickets to sell: 
Maximum of 8 Tickets can be sold per transaction\nEnter number of tickets to sell: 
Transaction canceled
Session terminated.
Enter session type <sales or admin>: 
Admin login successful.
Enter event name: 
Current number of tickets: 10\nEnter number of tickets to sell: 
10 tickets sold. New total: 0
Session terminated.